### PR TITLE
[vk] Add support for dailymotion embeds

### LINKED ISSUE #10661: [vk] Unable to extract vars (support for Dailymotion embeds)
### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2016.09.11.1*. If it's not read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2016.09.11.1**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through [README](https://github.com/rg3/youtube-dl/blob/master/README.md) and **most notably** [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [x] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

Add `-v` flag to **your command line** you run youtube-dl with, copy the **whole** output and insert it here. It should look similar to one below (replace it with **your** log inserted between triple ```):
```
youtube-dl --verbose https://vk.com/video-37468416_456239855
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--verbose', 'https://vk.com/video-37468416_456239855']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2016.09.11.1
[debug] Python version 3.4.2 - Linux-3.16.0-4-amd64-x86_64-with-LinuxMint-2-betsy
[debug] exe versions: ffmpeg 2.6.9, ffprobe 2.6.9, rtmpdump 2.4
[debug] Proxy map: {}
[vk] -37468416_456239855: Downloading webpage
ERROR: Unable to extract vars; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/youtube_dl/YoutubeDL.py", line 691, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/lib/python3.4/dist-packages/youtube_dl/extractor/common.py", line 347, in extract
    return self._real_extract(url)
  File "/usr/local/lib/python3.4/dist-packages/youtube_dl/extractor/vk.py", line 327, in _real_extract
    data_json = self._search_regex(r'var\s+vars\s*=\s*({.+?});', info_page, 'vars')
  File "/usr/local/lib/python3.4/dist-packages/youtube_dl/extractor/common.py", line 650, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
youtube_dl.utils.RegexNotFoundError: Unable to extract vars; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

### PR DESCRIPTION
**Pull request by kasper93**
### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

Adds Dailymotion embeds support for vk. 
Fixes #10661

### CODE REVIEW

File: youtube_dl/extractor/vk.py [hunk 1]
```diff
@@ -211,6 +212,25 @@ class VKIE(VKBaseIE):
...
+            # dailymotion embed
+            'url': 'https://vk.com/video-37468416_456239855',
+            'info_dict': {
+                'id': 'k3lz2cmXyRuJQSjGHUv',
+                'ext': 'mp4',
+                'description': 'Группа AniLibria.Tv В Контакте                 '
+                               '★★★ http://vk.com/anilibria ★★★                '
+                               'Перевод (полухардсаб): HighFive, Редактура: timo.'
+                               ',Оформление: Scarlet и Waspil  Озвучивание: Aemi,'
+                               ' Lali, Anzen & Ados (MVO) Тайминг: Quin',
```
Comment (dstftw) [inline youtube_dl/extractor/vk.py]: Replace with `md5:...`.

File: youtube_dl/extractor/vk.py [hunk 2]
```diff
@@ -211,6 +212,25 @@ class VKIE(VKBaseIE):
...
+                'title': 'Shokugeki no Souma: Ni no Sara TV-2 / Повар-Боец Сома '
+                'ТВ-2 - 11 серия | Aemi, Lali, Anzen & Ados (MVO) [AniLibria.Tv]',
```
Comment (dstftw) [inline youtube_dl/extractor/vk.py]: This as well.

File: youtube_dl/extractor/vk.py [hunk 3]
```diff
@@ -211,6 +212,25 @@ class VKIE(VKBaseIE):
...
+                'uploader': 'AniLibria.Tv',
+                'upload_date': '20160914',
+                'uploader_id': 'x1p5vl5',
+                'timestamp': 1473877246,
+            },
```
Comment (dstftw) [inline youtube_dl/extractor/vk.py]: Add:
```
'params': {
    'skip_download': True,
}
```

**Comment by kasper93** [comment]
Done. My bad forgot about "md5:" thingy.

File: youtube_dl/extractor/vk.py [hunk 4]
```diff
@@ -315,6 +333,12 @@ def _real_extract(self, url):
                 m_rutube.group(1).replace('\\', ''))
             return self.url_result(rutube_url)
 
+        dailymotion_urls = DailymotionIE._extract_urls(info_page)
+        if dailymotion_urls:
+            return self.playlist_result([
```
Comment (dstftw) [inline youtube_dl/extractor/vk.py]: It does not look like it's possible by design to embed several videos on a video page at vk. So just use `url_result` on the first URL.

Comment (kasper93) [inline youtube_dl/extractor/vk.py]: Ok, done.

**Comment by dstftw** [comment]
Cherry-picked in e3845525906228091fdf446f2cf2e9a20e93f59f, thanks.

**Pull request closed without merging**